uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/document-list.js

```diff
@@ -21,6 +22,10 @@ define([
 
       $scope.dueThisWeek = 0;
       $scope.dueToday = 0;
+
```

Comment:
Why the empty lines. Please remove them.

Reply:
removed!